wp-content/plugins/hurumap/api.php

```diff
@@ -308,13 +308,28 @@ function get_flourish_chart($request)
          * Add html2Canvas to allow the iframe to be downloadable
          * Add hostname so that we can access iframe document from subdomain
          */
-        $hostname = $request['origin'] ? $request['origin'] : $_SERVER['HTTP_HOST'];
+        $hostname = $request['origin'];
```

Comment:
Why are we getting `$hostname` from `$request['origin']`?... I could be wrong but does this mean if I send a request to the dashboard with `origin` kilemensi.com then it will set it to that? If so, then this is not what we should be doing... client shouldn't have that ability.

Reply:
@kilemensi yes, thats how it's set. It's been tough doing tests without being part of takwimu subdomain. Let's remove this once we resolve localhost testing issues. One being that I have no ACF fields to seed my local env. I have to rely on pushing to zeit.

Reply:
I hear you ... then lets add a PT story to track this issue because it's kind of a big deal @karimkawambwa (Including general clean up of the plugin e.g. I see we're still using `hurumap-data` as namespace when we should be using `hurumap` or better even version it `hurumap/v1`

Reply:
@kilemensi lets do this on another PR

> Including general clean up of the plugin e.g. I see we're still using hurumap-data as namespace when we should be using hurumap or better even version it hurumap/v1